templ_attr.cif

```diff
@@ -183,6 +182,70 @@ save_ms_index
     _type.contents               Integer
      save_
 
+save_q_coeff_element
+
+    _definition.update           2024-08-06
+    _description.text                   
+;
+     Element of the matrix _atom_site_Fourier_wave_vector.q_coeff.
+     Notation of the elements is identical to the one used in magCIF.
+     However, in magCIF the number of elements is limited to 3.
+;
+    _type.purpose                Number
+    _type.source                 Assigned
+    _type.container              Single
+    _type.contents               Integer
+    _enumeration.default         0
+save_
+
+save_q_coeff_seq_id
+
+    _definition.update           2024-07-26
+    _description.text                   
+;
+     Element of the matrix _atom_site_Fourier_wave_vector.q_coeff_seq_id.
+     Must match those given in _cell_wave_vector.seq_id.
+;
+    _name.linked_item_id         '_cell_wave_vector.seq_id'
+    _type.purpose                Link
+    _type.source                 Related
+    _type.container              Single
+    _type.contents               Integer
+save_
+
+save_site_ssg_symmetry
+
+    _definition.update           2024-12-12
+    _description.text                   
+;
+      The symmetry code of each atom site as the symmetry operation
+      number 'n' and the higher-dimensional translation 'm1...mp'.
+      These numbers are combined to form the code 'n m1...mp' or
+      n_m1...mp. The character string n_m1...mp is composed as follows:
+      'n' refers to the symmetry operation that is applied to the
+      superspace coordinates. It must match a number given in
+      _space_group_symop_ssg_id. 'm1...mp' refer to the translations
+      that are subsequently applied to the symmetry-transformed
+      coordinates to generate the atom used in calculating the angle.
+      These translations (t1,...tp) are related to (m1...mp) by the
+      relations m1=5+t1, ..., mp=5+tp. By adding 5 to the translations,
+      the use of negative numbers is avoided. The number 'p' must agree
+      with (_cell_modulation_dimension + 3). If there are no cell
+      translations, the translation number may be omitted. If no
+      symmetry operations or translations are applicable, then a single
+      full stop '.' is used.
+;
+    _type.purpose                Composite
+    _type.source                 Derived
+    _type.container              Single
+    _type.contents               Symop
+    loop_
+    _description_example.case
+    _description_example.detail
+    '.'                  'no symmetry or translation to site'          
+    '4'                  '4th symmetry operation applied'    
+    '7_645'              '7th symmetry position; +a on x, -b on y' 
```

Comment:
This example may be incompatible with the provided definition since the translation part consists of only 3 elements (p = 3).

The definition states that:
```
The number 'p' must agree with (_cell_modulation_dimension + 3). 
```
The `_cell_modulation_dimension` is currently defined to have values in the range of 1..8, thus leaving the minimal p value at 4.

Also, having an example with at least 4 translations would be useful to showcases the difference from the conventional 3D symmetry code data item.